example.env

```diff
@@ -6,3 +6,4 @@ PGADMIN_PASSWORD=
 JWT_SECRET=
 BCRYPT_SALT_ROUNDS=
 BCRYPT_SALT_TEST=
+REQUIRE_PASS_REDIS=
```

Comment:
hmm, kind of strange name - wouldn't just `REDIS_PASSWORD` be better?

Reply:
I used it because if I have worked on local Redis, I would have to pass `requirepass` in `redis.conf`. And in docs they always define this password with such keyword so i sticked with that.

Reply:
Oh, looking at the next comment, classic REDIS_PASSWORD is much more pleasing to the eye

Reply:
`--requirepass` is an argument that can be read as `Require password? If so, what should be the value?`

It's easier to understand what's the meaning of the value, when the env variable name is `REDIS_PASSWORD` than `REQUIRE_PASS_REDIS` 😉